neo/Consensus/ConsensusService.cs

```diff
@@ -440,11 +445,14 @@ private void OnStart(Start options)
                     CheckPreparations();
                     return;
                 }
+                InitializeConsensus(context.ViewNumber);
```

Comment:
if is not loaded, the default value must be 0, right? we can change the line 444 to
`InitializeConsensus(context.ViewNumber);`

Reply:
Exactly, if it is not load it is 0.

```cs
                InitializeConsensus(context.ViewNumber);
            }
            else
               InitializeConsensus(0);
```
There is an else below.

Reply:
Since the default value of `ViewNumber` is 0 it should be safe to avoid the else and do what @shargon suggests and reduce by 2 lines of code.

Reply:
I added the minor simplification suggested.

Reply:
Is not the context.viewnumber empty on startup if it was not loaded from db?

Reply:
Default value of byte is 0. It should be fine.